Dockerfile.rhel7

```diff
@@ -1,9 +1,9 @@
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13 AS builder
 WORKDIR  /go/src/github.com/openshift/oauth-proxy
 COPY . .
 RUN go build .
 
-FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+FROM registry.svc.ci.openshift.org/openshift/origin-v4.4:base
```

Comment:
I don't think there is such image, part of the reason this should be remove in a follow up

Reply:
interestingly enough, this appears in the .rhel7 image, let me change that

Reply:
good catch, I've switched these files in my mind accidentally